integ/test_init/test-guide.md

```diff
@@ -0,0 +1,125 @@
+###  Integration Test for Init Process of Fluent Bit on ECS
+
+Following this instruction, you will take the integration test for init process and get the test result
+
+
+
+#### Requirements
+
+1. [install AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+2. [set up AWS account credential](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-quickstart.html)
+3. [install AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html)
+
+
+
+#### Deploy test resources
+
+* create some Fluent Bit config files locally  (you can use any config files that meet the fluent bit config requirement)
```

Comment:
Let's capitalize all of these and make them all complete sentences. Also I'd say something more like "you can use any config files that are valid"